README.md

```diff
@@ -59,8 +59,17 @@ Example of `~/.ungitrc` configuration file to change default port and enable bug
 }
 ```
 
+
+SSH Authorization
+-----------------
+Ungit does run git commands in background and authentication will be asked continously when working with private repos.  Easiest way to prevent this is to set up ssh authorization and cache it.
+
+1. Follow github's [instruction](https://docs.github.com/en/github/authenticating-to-github/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent)
+2. Ensure `ssh-add -K ~/.ssh/id_rsa` or with whatever ssh key file name is executed.
```

Comment:
Is this not covered by the instructions linked above?

Additionally, it seems that the `-K` option only applies to (most) macs:
![image](https://user-images.githubusercontent.com/2564094/92332230-15eae200-f031-11ea-9b0a-7fadceb01ca1.png)

Reply:
It is covered, but not always followed, which is why I would like to ensure.

As far as `-K` being only for mac only I didn't know about. Maybe it be better to change to more targeted ssh-add